src/actions/vm.js

```diff
@@ -137,6 +140,31 @@ export function suspendVm ({ vmId }) {
   }
 }
 
+export function createVm ({ vm, transformInput = false, pushToDetailsOnSuccess = false }, { actionCorrelationId, ...actionPassThrough }) {
+  return {
+    type: CREATE_VM,
+    payload: {
+      vm,
+      transformInput,
+      pushToDetailsOnSuccess,
+    },
+    actionCorrelationId,
```

Comment:
I think for this purpose better create something like `metaPayload` in action.

Reply:
The `actionCorrelationId` is for the action itself, not for the payload.  When I read `metaPayload`, I read it as information about the payload itself.

AFAIK, there is no standard for redux action data beyond what we do in the app with putting the data to be worked on in the `payload` field.

Reply:
Okay. It can be `metaAction`, the point is to group data to make them more understandable.

Reply:
I did some research and it is more common to keep the action types clean and use the form:
````javascript
{
   type: CONSTANT,
   payload: { someData: true },
   meta: { correlationId: 123 },
}
````

As you mentioned, it does make the intention more explicit.  Additionally, since the technique being used to trace back the success/fail of an action via a `correlationId` can be used in a number of other places as well.  I'm updating the code to follow the `meta` field style in the action type.

Reply:
Moved to #724